android/app/src/main/AndroidManifest.xml

```diff
@@ -46,7 +46,7 @@
         android:exported="true"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|smallestScreenSize|uiMode"
         android:launchMode="singleTask"
-        android:windowSoftInputMode="adjustResize">
```

Comment:
so keyboard doesn't push up tab bar on android

Reply:
should take a look at how this impacts the send flow